db.init.js

```diff
@@ -0,0 +1,162 @@
+const mongoose = require('mongoose');
+const models = require('./models');
+
+const { Types } = mongoose;
+const { ObjectId } = Types;
+const {
+  Category, Order, OrderStage, Item, Filter, Discount,
+} = models;
+
+const dbConn = process.argv[2];
+
+const initialData = {
+  categories: [
+    {
+      _id: ObjectId('000000000000000000000000'),
+      title: 'Pizza',
+    },
+    {
+      _id: ObjectId('000000000000000000000001'),
+      title: 'Pasta',
+    },
+
+    {
+      _id: ObjectId('000000000000000000000002'),
+      title: 'Sandwiches',
+    },
+
+    {
+      _id: ObjectId('000000000000000000000003'),
+      title: 'Soup',
+    },
+
+    {
+      _id: ObjectId('000000000000000000000004'),
+      title: 'Salads',
+    },
+
+    {
+      _id: ObjectId('000000000000000000000005'),
+      title: 'Sides',
+    },
+
+    {
+      _id: ObjectId('000000000000000000000006'),
+      title: 'Deserts',
+    },
+
+    {
+      _id: ObjectId('000000000000000000000007'),
+      title: 'Drinks',
+    },
+  ],
+  orderStages: [
+    {
+      _id: ObjectId('000000000000000000000000'),
+      title: 'ordered',
+    },
+    {
+      _id: ObjectId('000000000000000000000001'),
+      title: 'baking',
+    },
+    {
+      _id: ObjectId('000000000000000000000002'),
+      title: 'finishing',
+    },
+    {
+      _id: ObjectId('000000000000000000000003'),
+      title: 'served',
+    },
+  ],
+  orders: [],
+  items: [
+    {
+      imgPath: '/ham_and_cheese.png',
+      price: 26.75,
+      title: 'Ham and cheese',
+      description: 'Ham and cheese ham and cheese ham and cheese ham and cheese ham and cheese',
+      filterIds: [],
+      categoryId: ObjectId('000000000000000000000000'),
+    },
+    {
+      imgPath: '/margarita.png',
+      price: 20.5,
+      title: 'Margarita',
+      description: 'Margarita margarita margarita margarita margarita margarita margarita',
+      filterIds: [],
+      categoryId: ObjectId('000000000000000000000000'),
+    },
+    {
+      imgPath: '/pepperoni.png',
+      price: 20.5,
+      title: 'Pepperoni',
+      description: 'Pepperoni pepperoni pepperoni pepperoni pepperoni',
+      filterIds: [],
+      categoryId: ObjectId('000000000000000000000000'),
+    },
+    {
+      imgPath: '/vegetable.png',
+      price: 17.3,
+      title: 'Vegetable',
+      description: 'Vegetable vegetable vegetable vegetable vegetable',
+      filterIds: [ObjectId('000000000000000000000000'), ObjectId('000000000000000000000001')],
+      categoryId: ObjectId('000000000000000000000000'),
+    },
+    {
+      imgPath: '/italian.png',
+      price: 28.0,
+      title: 'Italian',
+      description: 'Italian italian italian italian italian italian italian',
+      filterIds: [],
+      categoryId: ObjectId('000000000000000000000000'),
```

Comment:
@IgorChaikin Could you please add some `pasta` and `drinks` items to test categories switching